start.py

```diff
@@ -724,6 +760,99 @@ def _configure_kudu(deployment, cluster, kudu_version):
                                                                configs)
 
 
+def _setup_ssl(cluster, product_list, ssl_authentication_required):
+    _setup_ssl_ca_authority(cluster)
+    if not ssl_authentication_required:
+        for product in product_list:
+            _setup_ssl_encryption(cluster, product)
+    else:
+        for product in product_list:
+            _setup_ssl_encryption_authentication(cluster, product)
+
+
+def _setup_ssl_ca_authority(cluster):
+    ssl_ca_authority_commands = [
+        'mkdir -p {}'.format(SSL_SERVER_CONTAINER_DIR),
+        'mkdir -p {}'.format(SSL_CLIENT_CONTAINER_DIR),
+        'cd {}'.format(SSL_SERVER_CONTAINER_DIR),
+        ('openssl req -new -newkey rsa:4096 -days 365 -x509 -subj "/CN=Kafka-Security-CA"'
+         ' -keyout ca-private-key -out ca-public-key -nodes'),
+    ]
+    cluster.primary_node.execute(' && '.join(ssl_ca_authority_commands))
+
+
+def _setup_ssl_encryption(cluster, product):
```

Comment:
Same story here about referring to it as `service` instead of `product`

Reply:
Done.